src/app/pages/questions/components/question/question.component.html

```diff
@@ -122,8 +122,8 @@
         <radio-input
           *ngSwitchCase="'yesno'"
           [responses]="[
-            { code: '1', label: 'Yes' },
-            { code: '0', label: 'No' }
+            { code: 1, label: 'Yes' },
+            { code: 0, label: 'No' }
```

Comment:
would this break existing Qs? can we support both?

Reply:
Yeah you're right there are existing questionnaires with yesno questions that also use branching logic (and are using strings in the logic). It might be difficult to support both since we are immediately evaluating the logic against the answers, unless we populate the answers with both integer and string, which might not be a good idea. Should this be modified in the Redcap script then?

Reply:
I'll just undo this change in the meantime.

Reply:
I guess it is ok to keep it string for backwards compatibility. I was thinking supporting both was an option, but if not, then string would be fine.